OpenRA.Mods.Common/Util.cs

```diff
@@ -175,5 +175,23 @@ public static int RandomDelay(World world, int[] range)
 
 			return world.SharedRandom.Next(range[0], range[1]);
 		}
+
+		static WDist allMinimumRequiredScanRadius;
+		static WDist blockerMinimumRequiredScanRadius;
+		static Ruleset lastRules = null;
+
+		public static WDist MinimumRequiredScanRadius(Ruleset rules, ActorInfo ai, bool onlyBlockers = false)
+		{
+			if (lastRules != rules)
```

Comment:
I don't have the 'head-bandwidth' right now to figure out how to migrate this to ActorMap, so for now I've replaced the caching with a TODO.